app/views/facilities/index.html.haml

```diff
@@ -3,24 +3,40 @@
 
 .alert.alert-info= text(".welcome")
 
-- if @facilities.any?
+- if @azlist_on
+  .az_list_container
+    = render "azlist"
 
-  - if @recently_used_facilities.present?
-    %h4= text(".recently_used")
+- if @facilities.any?
+  - if not @azlist_on
+    - if @recently_used_facilities.present?
```

Comment:
Just thinking, maybe we can extract some partials so we have an az-list styled view and a non-az-list styled view. Then each one of those can be composed of calls to partials like `recent_products`, `recent_facilities`, and `full_list`. That way there's really only one top-level conditional and inside of each side of that conditional we can reorder and add/remove `<hr>`s as we want.

I'll reach out tomorrow, but I feel like maybe doing a little pairing might expedite the PR process.